app/views/events/show.html.erb

```diff
@@ -1,69 +1,77 @@
-<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+<!-- Event title -->
+<div class="bg-image">
```

Comment:
show view looks a little cheesy, I think it would be good to have it centered more with negative space on the sides. Here's a screenshot of my (relatively unstyled) index page for reference:
![image](https://user-images.githubusercontent.com/20048451/57575242-9ed11980-73fb-11e9-8163-2ed9eae51f34.png)